benchmarks/holoscan_flow_benchmarking/app_perf_graph.py

```diff
@@ -199,14 +218,56 @@ def main():
     if not args.live:
         filenames = args.filenames
         if len(filenames) == 0:
-            print("No filenames provided")
+            print("Error: No filenames provided")
             parser.print_help()
-            return
+            sys.exit()
         # check if the files exist
         for filename in filenames:
             if not os.path.isfile(filename):
-                print("File {} does not exist or is not a file.".format(filename))
-                return
+                print("Error: File {} does not exist or is not a file.".format(filename))
+                sys.exit()
+    else:
+        directory = args.filenames[0]
+        print(
+            "In live mode. The program will keep updating the graph with new \
+                performance data in the provided folder {}.\
+                Press Ctrl+C to stop.".format(
+                directory
+            )
+        )
+        if len(args.filenames) > 1:
+            print(
+                "\033[91mfilenames arguments has {} values. In live mode,\
+                    only one folder is acceptable. Provide one folder as filenames\
+                    in live mode. Exiting.\033[0m".format(
+                    len(args.filenames)
+                )
+            )
+            parser.print_help()
+            sys.exit()
+        # check if directory is a folder
+        if not os.path.isdir(directory):
+            print(
+                "\033[91mThe folder {} does not exist or is not a folder.\
+                    In live mode, a folder needs to be provided. Exiting.\033[0m".format(
+                    directory
+                )
+            )
+            parser.print_help()
+            sys.exit()
+
+    return args
+
+
+def print_message(message, silent):
+    if silent:
+        return
+    print(message)
+
+
+def create_dot_file(filenames, output_filename, live_graph, verbose, highlight, silent=False):
```

Comment:
We can keep using `verbose` instead of introducing another variable `silent`.

Reply:
@sohamm17 are we ok with updating some of the messages to print out only if verbose is true?  I was trying to keep the existing behavior but it makes more sense to not introduce the extra silent if that's not a concern

Reply:
@whom3 Yes, I think it is fine to skip printing a few things when `verbose` is not `true`.

Reply:
updated